backend/module/chaos/serverexperimentation/xds/xds_cache.go

```diff
@@ -136,11 +136,11 @@ func setSnapshot(resourceMap map[gcpTypes.ResponseType][]gcpTypes.ResourceWithTt
 	if isSnapshotEmpty {
 		scope.Tagged(map[string]string{
 			"cluster": cluster,
-		}).Counter("active_faults").Inc(-1)
+		}).Gauge("active_faults").Update(-1)
```

Comment:
Is this just going to set the gauge to -1? I think you need to track this value somewhere or compute it periodically

Reply:
Maybe add a test to make sure this works right?